actions/transaction_genesis_hashes.go

```diff
@@ -46,6 +47,12 @@ type transactionGenesisHashUpdateRes struct {
 
 // Creates a transaction.
 func (usr *TransactionGenesisHashResource) Create(c buffalo.Context) error {
+	start := time.Now()
+
+	defer func() {
```

Comment:
Does this need to be wrapped in a closure?

Reply:
Would you like to have here only function from service? In this function will be included histogram? But I dont know then how to work with defer function.